src/Analyser/NodeScopeResolver.php

```diff
@@ -3839,9 +3838,9 @@ private function processNodesForTraitUse($node, ClassReflection $traitReflection
 	}
 
 	/**
-	 * @return array{TemplateTypeMap, Type[], ?Type, ?Type, ?string, bool, bool, bool, bool|null}
+	 * @return array{TemplateTypeMap, Type[], ?Type, ?Type, ?string, bool, bool, bool, bool|null, bool, string|null}
 	 */
-	public function getPhpDocs(Scope $scope, Node\FunctionLike $functionLike): array
+	public function getPhpDocs(Scope $scope, Node $node): array
```

Comment:
Can we make this FunctionLike/Property union type? It can be native, it's gonna be downgraded automatically.